File: core/calc/summation/summation.js
var summation = function (num) {
  let sum = 0;

  for (let i = 1; i <= num; i++) {
    sum += i;
  }

  return sum;
};

// var summation = function (num) {
//   return (num * (num + 1)) / 2;
// };

console.log(summation(2)); // Outputs: 3
console.log(summation(8)); // Outputs: 36

File: core/calc/summation/summation.py
def summation(num):
    sum = 0
    for i in range(1, num + 1):
        sum += i
    return sum

# def summation(num):
#     return num * (num + 1) // 2

print(summation(2))  # Outputs: 3
print(summation(8))  # Outputs: 36
